application/classes/Ushahidi/Repository/Set.php

```diff
@@ -227,4 +227,19 @@ public function addPostToSet($set_id, $post_id)
 		// so that this is queued for the Notifications data provider
 		$this->emit($this->event, $set_id, $post_id);
 	}
+
+	/**
+	 * Gets the set names corresponding to the list of tag ids
+	 * @param $tag_ids
+	 * @return array
+	 */
+	public function getNamesByIds($sets_ids)
+	{
+		$result = $this->selectQuery(array('id' => $sets_ids))
+			->resetSelect()
+			->select('name')
+			->execute($this->db);
+		$result = $result->as_array('name');
```

Comment:
I only need the name so I used 'name' as the key ... I'm not sure if there's a better way to easily get an array like ['Tagg1', 'Tag random', 'Explosion tag'] other than getting the keys. Thoughts?

Reply:
@rjmackay tagging

Reply:
Try `as_array(null, 'name')` ? I'm not 100% sure it work but I think I've done it somewhere before